### ISSUE TITLE
humphrey.uwazi.io - language versions and system slow

### ISSUE THREAD

**Issue by HSips** [huridocs/uwazi#990]
The OAU/AU docs instance is rather slow. I recognise that we have uploaded very many documents and are working to curate them, just to note that the system is slow to respond. Maybe as we have larger and large collections, we may need to think about this.
Pretoria people also noted the speed issue.

I wonder whether having language versions can help optimise speed? The collection will need to be in English, French, Portuguese and Arabic

**Comment by kjantin** [comment]
May be addressed with next deployment

**Comment by daneryl** [comment]
@Hsips, can you give more detail on where is the system slow pls ? can not replicate in 4 different devices with firefox/chrome and a really slow one and its working fine for me.
Large collection should not affect performance for now unless we are talking about millions of documents.

**Comment by HSips** [comment]
Thanks for this @daneryl 
I was adding metadata to uploads and publishing..
Will try again and see.
Thanks

**Comment by daneryl** [comment]
@HSips is this and #991  solved for you ? or still having problems ?

**Comment by HSips** [comment]
Thanks for this @daneryl To answer:
1. System slows down intermittently. When this happens, I get the 'unresponsive script' message.
2. I am unable to view the documents I uploaded in October 2016. I have tried to create 2 temporary document types called 'AU PRC evacuate' and AU Assembly evacuate' to move them into public library and hopefully view them from there but still. It keeps loading for hours.
3. No. we still do not have the language versions.

Sorry for this omnibus response. Maybe these are technically 3 different things.
